09-Multiparticle-Collision-Dynamics/02-Diffusion-in-Solution.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    sim.run(200000, write_at_start=True)
</pre><p>Remove <code>write_at_start=True</code>. Tutorials should promote the use of default behavior. Users can customize their own scripts as desired.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>